date/date_c.php

```diff
@@ -218,7 +218,7 @@ public static function getLastErrors() {}
      * @link https://secure.php.net/manual/en/datetimeimmutable.modify.php
      * @param string $modifier  <p>A date/time string. Valid formats are explained in
      * {@link https://secure.php.net/manual/en/datetime.formats.php Date and Time Formats}.</p>
-     * @return static
+     * @return static|false
```

Comment:
maybe you can add a comment about on what condition it returns false

Reply:
Should I move the PHP doc line just below onto this line? Wasn't sure whether that was done or if I should keep it separate?

Reply:
You can do like `* @return static|false False is returned ...` but avoid `* @return static|false is returned ...`
`* @return static|falseis returned ...` is obviously invalid